tests/effective_dps_test.cpp

```diff
@@ -527,6 +527,7 @@ static void check_knives( const std::function<Approx( const std::string & )> &ca
         CHECK( calc_expected_dps( "tanto" ) == 15.78 );
         CHECK( calc_expected_dps( "kris" ) == 14.08 );
         CHECK( calc_expected_dps( "knife_rambo" ) == 14.36 );
+        CHECK( calc_expected_dps( "knife_rm42" ) == 39.13 );
```

Comment:
Keep the comment around, please.
Doing the math, `39.13 / ( 1.3 * 1.2) ~= 25`, it's still too high.
```suggestion
        CHECK( calc_expected_dps( "knife_rm42" ) == 39.13 );
```

Reply:
I don't know why the DPS test is overreporting it so much, or rather why the test is underreporting the combat knife by comparison. A CVD'd combat knife would have 34 damage, so this shouldn't be weaker than that. I will increase its weight more and see what that does.

Reply:
formula for moves per attack = 65 + (4 per each 250ml of volume) + (1 per 60 grams of weapon weight)

Reply:
OK, as is it is slower than the combat knife being longer and heavier, it has +1 to hit due to being SHORT and not HAND length, and its cut damage is exactly 30% higher than a combat knife (CVD property) so why exactly is its DPS over twice as high?

Reply:
i think it has to do with piercing damage on low damage rapid strike weapons having trouble with the soldier zombie and the combat knife sits fairly close to the sweetspot to basically "cheese"  that part of the test. low accuracy probably makes it perform poorly vs the smoker as well